src/services/shims.ts

```diff
@@ -322,8 +323,19 @@ module ts {
         }
     }
 
+    function serializeResultAsJson(target: Shim, key: string | symbol, descriptor: PropertyDescriptor) {

```

Comment:
Yes, but not for this function. Does it give you an error if you don't handle symbols as well?